imports/client/components/NotificationCenter.tsx

```diff
@@ -364,7 +370,7 @@ type NotificationCenterTracker = {
 
 const NotificationCenter = () => {
   const tracker: NotificationCenterTracker = useTracker(() => {
-    const canUpdateGuesses = Roles.userHasPermission(Meteor.userId(), 'mongo.guesses.update');
+    const canUpdateGuesses = deprecatedIsActiveOperator(Meteor.userId());
```

Comment:
`pendingGuesses` might need to be restructured or made to work for non-operators or scoped to specific hunts or something to make this work reasonably in a tracking-operator-per-hunt world.